config/jobs/kubernetes/sig-windows/windows-gce.yaml

```diff
@@ -180,6 +185,124 @@ periodics:
   annotations:
     testgrid-dashboards: google-windows
     testgrid-tab-name: windows-gce-serial
+
+- name: ci-kubernetes-e2e-windows-containerd-gce
+  decorate: true
+  extra_refs:
+  - org: yujuhong
+    repo: gce-k8s-windows-testing
+    base_ref: master
+    path_alias: k8s.io/gce-k8s-windows-testing
+  interval: 2h
+  labels:
+    preset-k8s-ssh: "true"
+    preset-service-account: "true"
+    preset-e2e-gce-windows: "true"
+    preset-e2e-gce-windows-containerd: "true"
+  spec:
+    containers:
+    - command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --check-leaked-resources
+      - --cluster=
+      - --extract=ci/k8s-master
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=8
+      - --provider=gce
+      - --gcp-nodes=2
+      - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/gce-k8s-windows-testing/run-e2e.sh
+      - --test-cmd-args=--ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\] --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|\[Feature:.+\] --minStartupPods=8
+      - --test-cmd-args=--node-os-distro=windows
+      - --timeout=120m
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20191201-9ade087-master
+  annotations:
+    fork-per-release: "true"
+    fork-per-release-replacements: "--extract=ci/k8s-master -> --extract=ci/k8s-beta"
+    testgrid-dashboards: google-windows, sig-windows, sig-release-master-informing, sig-node-containerd
+    testgrid-tab-name: gce-windows-containerd-master
+    description: Runs tests on a Kubernetes cluster with Windows containerd nodes on GCE
+
+- name: ci-kubernetes-e2e-windows-containerd-gce-alpha-features
```

Comment:
What alpha features does containerd depend on?

Reply:
No. I think this is to validate whther kubernetes alpha features work with containerd.

Reply:
Ok. I'm not sure that we need the alpha and serial test jobs w/ containerd yet, but I also don't know if there's much harm in adding them now. @yliaog what do you think?

Reply:
OK. Let's add the most basic one first. :)